index.d.ts

```diff
@@ -1,2 +1,2 @@
-export declare function combo(...shortcuts: string[]): any;
+export declare function combo(...shortcuts: (string|boolean)[]): any;
```

Comment:
Although not technically correct, this is probably the only way in ts.

@bigopon is there a way in TS to define something like this?:
```ts
function combo(...shortcuts: string[], runsInInput?: boolean): any;
```
The last argument is an optional boolean.

Reply:
well, I remember that TS doesn't allow this. and just confirmed again
![image](https://user-images.githubusercontent.com/9994529/79170724-8e0efc00-7e33-11ea-8f0c-40143d0800b8.png)
It's from JS I believe